docs/source/installation.rst

```diff
@@ -43,10 +44,10 @@ Mac OS X
 Linux
     Use your package manager to install ``numpy`` and ``scipy``
         Ubuntu: ``sudo apt-get install python-numpy python-scipy``
-    Run ``pip install pint``
+    Run ``pip install pint`` (for system python installs, use ``sudo pip``)
```

Comment:
I haven't seen much documentation using `--user`, but on the other hand I kinda wish more people did.